designs/2022-community-eslint-org/README.md

```diff
@@ -0,0 +1,328 @@
+
+I think all of [@aladdin-add](https://github.com/aladdin-add),
+[@ota-meshi](https://github.com/ota-meshi),
+[@voxpelli](https://github.com/voxpelli) & myself would be in for helping to
+maintain at least the following packages:
+[`@mysticatea/eslint-plugin`](https://github.com/mysticatea/eslint-plugin),
+[`eslint-plugin-eslint-comments`](https://github.com/mysticatea/eslint-plugin-eslint-comments),
+[`eslint-plugin-es`](https://github.com/mysticatea/eslint-plugin-es),
+[`eslint-plugin-node`](https://github.com/mysticatea/eslint-plugin-node),
+[`eslint-utils`](https://github.com/mysticatea/eslint-utils) &
+[`regexpp`](https://github.com/mysticatea/regexpp) (all from
+[@mysticatea](https://github.com/mysticatea)) &
+[`eslint-plugin-promise`](https://github.com/xjamundx/eslint-plugin-promise)
+(which is used a lot as well and was mentioned by the current maintainer
+([@xjamundx](https://github.com/xjamundx)) that he would like to transfer it to
+the community org if one was going to exist).
+
+### What would be the acceptance criteria?
+
+I personally think that the acceptance criteria for a package to go under the
+`@eslint-community` org would be:
+
+- Is it a package that is ESLint-related?  
+  Mostly this will be ESLint plugins, but I can see (unmaintained) dependencies
+  of such packages, closely related packages or packages split from the main
+  ESLint repo (like
+  [`eslint-formatter-codeframe`](https://github.com/fregante/eslint-formatter-codeframe)
+  or
+  [`eslint-formatter-table`](https://github.com/fregante/eslint-formatter-table),
+  which are currently maintained by [@fregante](https://github.com/fregante)) or
+  used by the main repo (like
+  [`eslint-utils`](https://github.com/mysticatea/eslint-utils) &
+  [`regexpp`](https://github.com/mysticatea/regexpp)) to be in the
+  `@eslint-community` org as well.
+
+- Is it widely depended upon throughout the community?  
+  Don't have an exact number in mind, but all mentioned packages have at least
+  3M downloads/week.  
+  Only exceptions are `eslint-plugin-eslint-comments` (1M downloads/week) &
+  `@mysticatea/eslint-plugin` (1.3K downloads/week), which is used as a
+  devDependency for all other mentioned packages by `@mysticatea`.
+
+- It didn't receive an update in the last 12(?) months  
+  I think we can make an exception for packages that did get a PR merged which
+  was created by a member of the new org's "core" team (mostly for updating
+  compatibility with newer versions of ESLint).
+
+  We can also make an exception for packages that meet the other criteria, but
+  where the owner want to transfer ownership to the new org.  
+  I can for instance imagine that
+  [@not-an-aardvark](https://github.com/not-an-aardvark) would want to transfer
+  ownership of
+  [`eslint-plugin-eslint-plugin`](https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin)
+  to the new org as well, as it's a recommended plugin for linting custom
+  plugins.  
```

Comment:
👍 I had chatted this with not-an-aardvark a while ago, we agreed to transfer the repo to a better home.